### PR TITLE
Make addition function

### PR DESCRIPTION
**Pull request by JakeRoggenbuck**

### CODE REVIEW

File: examples/adding_function.cz [hunk 1]
```diff
@@ -0,0 +1,13 @@
+~Prompt "First number: " to and set input a~
+int a = sys.read("First number: ")
+~Prompt "Second number:" to and set input b~
+int b = sys.read("Second number: ")
+
+~Define add function~
+fun add(fnum, snum){
+	~Writes the sum of a and b to console as stdout~
```
Comment (adamhutchings) [inline examples/adding_function.cz]: Outdated comment

File: examples/adding_function.cz [hunk 2]
```diff
@@ -0,0 +1,13 @@
+~Prompt "First number: " to and set input a~
+int a = sys.read("First number: ")
+~Prompt "Second number:" to and set input b~
+int b = sys.read("Second number: ")
+
+~Define add function~
+fun add(fnum, snum){
```
Comment (adamhutchings) [inline examples/adding_function.cz]: Also, declare types of variables with `fun add(int fnum, int snum) -> int`

Comment (JakeRoggenbuck) [reply examples/adding_function.cz]: Got it

**Review by adamhutchings — APPROVED** [review]
Looks good!

**Pull request merged**